macOS/LocalPackages/NetworkProtectionMac/Sources/VPNNotifications/Resources/de.lproj/Localizable.strings

```diff
@@ -0,0 +1,36 @@
+/* The body of the notification when Privacy Pro subscription expired */
+"vpn.entitlement.expired.notification.body" = "Abonniere DuckDuckGo, um die Verbindung zum DuckDuckGo-VPN wiederherzustellen.";
```

Comment:
Not sure what happened here, how come we ended up with new VPN strings?

Reply:
Maybe some untranslated string that was included in my .xliff?